src/components/sub/menu/menu-item.jsx

```diff
@@ -1,23 +1,19 @@
-import React from 'react'
-import remixiconUrl from 'remixicon/fonts/remixicon.symbol.svg'
```

Comment:
Removing this import is part of the fix. It wasn't playing well or is not imported in a way that webpack can work with. Idk exactly but removing it keeps the compiler from complaining.